go/tasks/aws/config.go

```diff
@@ -18,22 +21,21 @@ const ConfigSectionKey = "aws"
 
 var (
 	defaultConfig = &Config{
-		Region:               "us-east-1",
-		MaxErrorStringLength: 150,
-		Retries:              3,
-		CatalogCacheTimeout:  config.Duration{Duration: time.Second * 5},
+		Region:  "us-east-2",
+		Retries: 3,
 	}
 
 	configSection = pluginsConfig.MustRegisterSubSection(ConfigSectionKey, defaultConfig)
 )
 
 // Config section for AWS Package
 type Config struct {
-	Region               string          `json:"region" pflag:",AWS Region to connect to."`
-	AccountID            string          `json:"accountId" pflag:",AWS Account Identifier."`
-	Retries              int             `json:"retries" pflag:",Number of retries."`
-	MaxErrorStringLength int             `json:"maxErrorLength" pflag:",Maximum size of error messages."`
-	CatalogCacheTimeout  config.Duration `json:"catalog-timeout" pflag:"\"5s\",Timeout duration for checking catalog for all batch tasks"`
+	Region               string            `json:"region" pflag:",AWS Region to connect to."`
+	AccountID            string            `json:"accountId" pflag:",AWS Account Identifier."`
+	Retries              int               `json:"retries" pflag:",Number of retries."`
+	LogLevel             aws.ClientLogMode `json:"logLevel" pflag:"-,Defines the Sdk Log Level."`
+	MaxErrorStringLength int               `json:"maxErrorLength" pflag:",Maximum size of error messages."`
+	CatalogCacheTimeout  config.Duration   `json:"catalog-timeout" pflag:"\"5s\",Timeout duration for checking catalog for all batch tasks"`
```

Comment:
this is interesting that we add catalogcache timeout in aws config no?

Reply:
This is nonsense :-D I don't know when did this slip in.. it existed before this PR... it's not used/referenced anywhere... I meant to delete it...